runtime/src/main/scala/akka/grpc/internal/AkkaHttpClientUtils.scala

```diff
@@ -197,8 +196,9 @@ object AkkaHttpClientUtils {
                                 .map[akka.grpc.javadsl.Metadata](h => new JavaMetadataImpl(new HeaderMetadataImpl(h)))
                                 .toJava
                           }))
-                    case Default(_, _, _) => throw mapToStatusException(response, Seq.empty)
-                    case e                => throw new IllegalStateException(s"Expected chunked or default response but got $e")
+                    case _ =>
+                      response.entity.discardBytes
```

Comment:
Can't we use `()` here? as we're calling this for its side-effect

```suggestion
                      response.entity.discardBytes()
```